Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.13.15 AS containerrun
+
+ENV GO111MODULE on
+
+RUN CGO_ENABLED=0 go build -ldflags="-s -w" -o "/usr/local/bin/container-run" code.cloudfoundry.org/quarks-container-run/cmd
+
+################################################################################
+FROM scratch
```

Comment:
Nit: Maybe it's worth to make it like in the quarks-operator so it is possible to swap with a base image? https://github.com/cloudfoundry-incubator/quarks-operator/blob/master/Dockerfile#L29

Reply:
Isn't scratch even better? You can build your own image and just

`COPY --from=cfcontainerization/containerrun /usr/local/bin/container-run /usr/local/bin/container-run`